src/Fixer/ClassNotation/SelfAccessorFixer.php

```diff
@@ -178,13 +178,11 @@ private function getClassStart(Tokens $tokens, $index, $namespace)
         foreach (array_reverse(Preg::split('/(\\\\)/', $namespace, -1, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE)) as $piece) {
             $index = $tokens->getPrevMeaningfulToken($index);
             if ('\\' === $piece) {
-                if (!$tokens[$index]->isGivenKind([T_NS_SEPARATOR])) {
-                    return null;
-                }
-            } else {
-                if (!$tokens[$index]->equals([T_STRING, $piece], false)) {
+                if (!$tokens[$index]->isGivenKind(T_NS_SEPARATOR)) {
                     return null;
                 }
+            } elseif (!$tokens[$index]->equals([T_STRING, $piece], false)) {
```

Comment:
Shouldn't this be handled by `no_superfluous_elseif`? Maybe worth adding a test case?

Reply:
the problem is with in the `elseif` - fixer doesn't fix;
```php
<?php
if ($foo) {

} else {
    if ($b) {
        return;
    }
}
```